avocado/utils/network/interfaces.py

```diff
@@ -128,6 +128,65 @@ def add_ipaddr(self, ipaddr, netmask):
         except Exception as ex:
             raise NWException("Failed to add address {}".format(ex))
 
+    @property
+    def vlans(self):
+        list_of_vlan_ids = {}
+        for i in self.host.interfaces:
```

Comment:
This method is part of *a* specific interface, so IMO makes no sense to get vlans of *other* interfaces.

What I'm missing?

Reply:
I came at this using the `/proc/net/vlan/config` file. Let me know what you think.